tools/quickstart/quickstart/docker.py

```diff
@@ -110,19 +156,68 @@ def update_and_start(host_base_dir: str) -> None:
     click.echo("\nAll services are running. Congratulations!")
 
 
-def wait_for_chain_spec() -> None:
-    while not os.path.exists(SHARED_CHAIN_SPEC_PATH):
+def wait_for_chain_spec(base_dir) -> None:
+    while not os.path.exists(os.path.join(base_dir, SHARED_CHAIN_SPEC_PATH)):
         time.sleep(1)
 
 
-def get_optional_docker_service_names() -> List[str]:
+def get_optional_docker_service_names(base_dir) -> List[str]:
     docker_service_names = []
 
-    if is_netstats_prepared():
+    if is_netstats_prepared(base_dir=base_dir):
         docker_service_names.append("netstats-client")
 
-    if is_bridge_prepared():
+    if is_bridge_prepared(base_dir=base_dir):
         docker_service_names.append("bridge-client")
         docker_service_names.append("mainnet-node")
 
     return docker_service_names
+
+
+def does_docker_compose_file_exist(base_dir):
+    return os.path.isfile(
+        os.path.join(base_dir, "docker-compose.yaml")
+    ) or os.path.isfile(os.path.join(base_dir, "docker-compose.yml"))
```

Comment:
Maybe slightly more readable:

```Py
filenames = [os.path.join(base_dir, "docker-compose." + extension) for extension in ["yml", "yaml"]]
return any(os.path.isfile(filename) for filename in filenames)
```

Reply:
I removed handling both extensions, as it would have not worked properly anyways with the new copying.